Library/Formula/bazaar.rb

```diff
@@ -20,11 +20,8 @@ def install
     system "make", "man1/bzr.1"
     man1.install "man1/bzr.1"
 
-    # Put system Python first in path
-    ENV.prepend_path "PATH", "/System/Library/Frameworks/Python.framework/Versions/Current/bin"
-
     system "make"
-    inreplace "bzr", "#! /usr/bin/env python", "#!/usr/bin/python"
+    inreplace "bzr", "#! /usr/bin/env python", "#!#{which "python"}"
```

Comment:
We don't want `/usr/bin/env python` in the case of linux either, for the same reason that it's not wanted in machomebrew; it's too often that a entirely different python is on the path than the one we compiled with. We should pin it down to the one that's on the path during build.